### PR TITLE
[ECO-246] Rebranding for component SDK

### PR DESCRIPTION
**Pull request by micaelbergeron**
This PR rebrands the SDK

Notable changes are:

- Mural icon changed to the new symbol
- Mural logo changed to the new wordmark
- `coral` color replaced by `red-70`
- Reworded `MURAL` for `Mural` across the SDK

### Screenshots

*The picker gets a new color on the buttons*

image

*Account chooser new logos*

image

### CODE REVIEW

File: packages/mural-account-chooser/src/components/account-chooser/index.tsx [hunk 1]
```diff
@@ -314,6 +310,11 @@ const AccountChooser: React.FC<AccountChooserPropTypes> = (
       ACCOUNT_CHOOSER_ACTION.NEW_ACCOUNT,
     );
 
+  const Logo = theme === 'light' ? MuralLogo : MuralLogoMono;
+  Logo;
+
+  console.log(MuralLogo, MuralLogoMono);
```
Comment (KavenTheriault) [inline packages/mural-account-chooser/src/components/account-chooser/index.tsx:316]: Don't forget to remove this line

Comment (micaelbergeron) [reply packages/mural-account-chooser/src/components/account-chooser/index.tsx]: Done, thanks eagle-eyed-man.

Comment (micaelbergeron) [reply packages/mural-account-chooser/src/components/account-chooser/index.tsx]: :eagle:

File: storybook/stories/mural-account-chooser.stories.tsx [hunk 2]
```diff
@@ -1,17 +1,25 @@
...
-import { ACCOUNT_CHOOSER_ACTION } from '../packages/mural-account-chooser/dist';
-import AccountChooser, {
-  AuthorizeParams,
-} from '../packages/mural-account-chooser/src/components/account-chooser';
+import {
+  AccountChooser,
+  ACCOUNT_CHOOSER_ACTION,
+} from '@muraldevkit/mural-integrations-mural-account-chooser';
+
+// TODO: Remove this
```
Comment (btuohy-mural) [inline storybook/stories/mural-account-chooser.stories.tsx:7]: 👀

Comment (btuohy-mural) [reply storybook/stories/mural-account-chooser.stories.tsx:7]: Did you mean to say "Remove this before merging the PR" or "Remove this eventually...."?

**Pull request merged**